Components/Avatar/avatar.html

```diff
@@ -0,0 +1,81 @@
+ <!DOCTYPE html>
+ <html lang="en">
+ <head>
+     <meta charset="UTF-8">
+     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <title>Avatar</title>
+
+     <link rel="stylesheet" href="avatar.css">
+
+     <!-- <link rel="stylesheet" href="landingPage.css"> -->
+ </head>
+ <body>
+
+    <hr>
+<section class="avatar-component component">
+
+<div class="component-info">
+    <div class="component-heading">Avatar</div>
+
+    <p class="info-text">Avatar component is used to show user's profile 
+        picture on profile information page, on navigation bar, in blogs grid items.</p>
+
+<div class="component-subHeading">Image Avatar</div>
+
+<p class="info-text">Avatar is available in 4 different sizes. You can use image in Avatar. You need to include
+     these classes in the image tag.Choose the appropriate classes according to the required size <b> img-avatar xl, img-avatar l, img-avatar m, img-avatar s (e.g. class="img-avatar m")</b>
+    Don't forget to add Responsive Image class names for img element.</p>
+
+
+</div>
+
+<div class="component-display-container" id="image-avatar-component">
+
+<img class="img-avatar xl" src="https://media.istockphoto.com/photos/lion-predator-dangerous-mane-picture-id1287214055?s=612x612" alt="avatar image">
```

Comment:
Can Use Prettier for Formatting / Beautifying Code (Space from the left side for a child element)